less/v2/dark/typography.less

```diff
@@ -28,10 +27,12 @@
 
   .roll-link > i.fa-dice-d20, a.inline-roll > i.fa-dice-d20 { filter: invert(1); }
 
-  .fvtt:is(.advice, .narrative, .quest) { background: var(--dnd5e-color-blue-gray-4); }
-
   .notable {
-    &::before, &::after { filter: grayscale(1); }
+    --dnd5e-block-background: var(--dnd5e-color-blue-gray-4);
+    --dnd5e-block-border-color: #212121;
+    // TODO this filter is not applying to the svg elements.
```

Comment:
Do you have an example of the issue this causes? Should we apply the filter anyway, even if it doesn't affect SVGs?

Reply:
It was more the lack of change because the svg element is black...maybe my comment there wasn't as precise as it could have been.

Mainly, it appeared as if it was a superfluous rule. If a user wanted to supply a different svg or image, the assumption was that a grayscale filter would likely also be removed by the user (or an alternative filter added due to their specific image needs).

Reply:
Keep it. I think I vaguely remember this, and I thought the red tinge was noticeable enough at the time.

Reply:
Re-added in 98ce9dc